recipe/meta.yaml

```diff
@@ -41,12 +40,12 @@ outputs:
       requires:
         - pip
 
-  - name: py-rattler
+  - name: py_rattler
     build:
       noarch: generic
     requirements:
       run:
-        - {{ pin_subpackage('py_rattler', max_pin="x.x.x") }}
+        - {{ pin_subpackage('py-rattler', exact=True) }}
```

Comment:
This will break as it will only be a single instance of `py_rattler` that is dependent on exactly the architecture-specific `py-rattler` in the fastest build. Either keep the loose pin or remove the `noarch`.